### ISSUE TITLE
pdflatex: Misplaced omit for markdown table

### ISSUE THREAD

**Issue by braggpeaks** [doxygen/doxygen#6390]
Tables like

```
| header1 | header2 |
| -- | -- |
| content1 | content2 |
```

throw error (which can be acknowledged):
```
! Misplaced \omit.
\multispan ->\omit
                   \@multispan
l.67 \end{longtabu}
```

This has obviously been introduced with 1.8.14, as 1.8.12 still works with the same table.
Running on most current Windows 10 with most current MikTeX packages.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.8.15.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).

**Comment by braggpeaks** [comment]
Tested with 1.8.15, resolved. Thank you very much.
